### PR TITLE
Docs: Default value of table level distribution-mode should be not set

### PR DESCRIPTION
**Pull request by manuzhang**

### CODE REVIEW

File: docs/docs/configuration.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ Iceberg tables support table properties to configure table behavior, like the de
...
 | write.target-file-size-bytes                         | 536870912 (512 MB)          | Controls the size of files generated to target about this many bytes                                                                                                                              |
 | write.delete.target-file-size-bytes                  | 67108864 (64 MB)            | Controls the size of delete files generated to target about this many bytes                                                                                                                       |
-| write.distribution-mode |  none, see engines for specific defaults, for example [Spark Writes](spark-writes.md#writing-distribution-modes) | Defines distribution of write data: __none__: don't shuffle rows; __hash__: hash distribute by partition key ; __range__: range distribute by partition key or sort key if table has an SortOrder |
-| write.delete.distribution-mode                       | hash                        | Defines distribution of write delete data                                                                                                                                                         |
```
[the remark this answers is not in the document]
Comment (manuzhang) [reply docs/docs/configuration.md:71]: However, it's also used for Flink write, whose default value is `none`. I think it should reflect the current status.

[the remark this answers is not in the document]
Comment (nastra) [reply docs/docs/configuration.md:71]: yes it should reflect the current status. What I'm saying is that we should document what the default is for Spark (on the spark page) while we're already modifying these properties

[the remark this answers is not in the document]
Comment (manuzhang) [reply docs/docs/configuration.md:71]: It's already documented.

https://github.com/apache/iceberg/blob/bfeaaeb8181bf035367761a0c6c907055f4f00cf/docs/docs/spark-writes.md?plain=1#L397

**Comment by manuzhang** [comment]
@nastra any more comments?

**Review by nastra — APPROVED** [review]

**Comment by Fokko** [comment]
Thanks for fixing this @manuzhang and thanks for the review @nastra

**Pull request merged**